crates/ln-dlc-node/src/ln/mod.rs

```diff
@@ -22,3 +22,10 @@ pub(crate) use logger::TracingLogger;
 ///
 /// This constant only applies to the coordinator.
 pub(crate) const JUST_IN_TIME_CHANNEL_OUTBOUND_LIQUIDITY_SAT: u64 = 200_000;
+
+/// When handling the [`Event::HTLCIntercepted`], the user might not be online right away. This
+/// could be because she is funding the wallet through another wallet. In order to give the user
+/// some time to open 10101 again we wait for a bit to see if we can establish a connection.
+///
+/// This constant specifies the amount of time (in seconds) we are willing to delay a payment.
+pub(crate) const HTLC_INTERCEPTED_CONNECTION_TIMEOUT: u64 = 10;
```

Comment:
I would go with 30 seconds, 10s is pretty tricky to make it on time when you're trying to open another app / website.

Reply:
I can increase it, but I am not sure if the other hops will simply timeout before us then.

Reply:
They won't timeout that fast. Afaik the HTLC will be added already to your node and the timeout is related to the CTLV timout, so actually quiet high.